docs/yaml/elements/lat_lng.yaml

```diff
@@ -3,23 +3,6 @@ csv-header-name: lat_long
 description: |-
   The latitude and longitude of a polling location in `WGS 84`_ format. Both
   latitude and longitude values are measured in decimal degrees.
-post: |-
```

Comment:
Why was this block removed?

Reply:
This `PollingLocation` XML example post was relocated from the lat_long.yaml to the [polling_location.yaml](https://github.com/votinginfoproject/vip-specification/blob/ba3eaa2a0dd359ea663c317f21697a3dfd29292a/docs/yaml/elements/polling_location.yaml#L18) file.